### DISCUSSION TITLE
Input text length limit too low for data feed to script step.

### DISCUSSION THREAD

**Discussion by halfong** [langgenius/dify#5455] [Help]
Say we got a data to input and handled by a **Custom Script (not AI)** to extract the info.
There's no need to set limit so low for data which used by a script, right ?

<img width="748" alt="image" src="https://github.com/langgenius/dify/assets/2819116/a698c000-fbdf-4a8a-9e94-361cd8fa3984">

**Comment by crazywoola** [comment]
It will save into database and database has a limitation of that column.
